functorch/functorch/_src/partitioners.py

```diff
@@ -333,7 +338,7 @@ def ban_recomputation(node):
             # then we don't allow recomputation.
             if 'tensor_meta' not in node.meta:
                 return False
-            input_tensors_size = sum(_size_of(i.meta['tensor_meta']) for i in node.args if isinstance(i, fx.Node))
+            input_tensors_size = sum(_maybe_size_of(i) for i in node.args if isinstance(i, fx.Node))
             output_size = _size_of(node.meta['tensor_meta'])
```

Comment:
does this node.meta always have a `tensor_meta` - or does it need the same treatment as above, `_maybe_size_of`?